src/main/java/uk/gov/hmcts/reform/unspec/validation/interfaces/WitnessesValidator.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.unspec.validation.interfaces;
+
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.model.dq.DQ;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static uk.gov.hmcts.reform.unspec.enums.YesOrNo.YES;
+
+public interface WitnessesValidator {
+
+    default CallbackResponse validateRespondentDqWitnesses(CallbackParams callbackParams) {
+        return validateWitnesses(callbackParams.getCaseData().getRespondent1DQ());
+    }
+
+    default CallbackResponse validateApplicantDqWitnesses(CallbackParams callbackParams) {
+        return validateWitnesses(callbackParams.getCaseData().getApplicant1DQ());
+    }
+
+    private CallbackResponse validateWitnesses(DQ dq) {
```

Comment:
probably we can just make this as default method and pass applicant or respondent DQ object in the calling classes. we don't need the above two default methods.

Reply:
Good idea, I'm going to raise as a tech debt, we can probably do the same for the Experts

Reply:
Made here: https://tools.hmcts.net/jira/browse/CMC-1568